sagemaker_model_monitor/fairness_and_explainability_jsonlines/SageMaker-Monitoring-Bias-Drift-for-Endpoint.ipynb

```diff
@@ -0,0 +1,1949 @@
+{
```

Comment:
<p>How about using inline code here?</p><ul><li>Notice <strong>the value of the </strong><code><strong>data</strong></code><strong> field in </strong><code><strong>groundTruthData</strong></code><strong>  must be in the same format as how the ground truth labels are stored in the input dataset</strong>.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Thanks, I will revise it to,</p><p> </p><ul><li>Notice the value of the <code>data</code> field in <code>groundTruthData</code><strong> must be in the same format as how the ground truth labels are stored in the input dataset</strong>.</li></ul><p> </p><p>----</p><p> </p><p>For your information: I wanted to highlight the entire sentence but then inline code tag doesn't work well within the bold tag. On the (readthedocs) web page it shows as </p><p> </p><ul><li>Notice <strong>the value of the <code></code>data<code></code> field in <code></code>groundTruthData<code></code> must be in the same format as how the ground truth labels are stored in the input dataset</strong>.</li></ul><p> </p><p>In the new revision the bold tag start from "must be..." to avoid the conflict.</p>